frontend/src/components/Contexts/EmergencyContext.tsx

```diff
@@ -0,0 +1,78 @@
+import { createContext, FC, useContext, useEffect, useState } from 'react'
+import { useRobotContext } from './RobotContext'
+import { AlertType } from './AlertContext'
+import { DockAlertContent, DockAlertListContent } from 'components/Alerts/DockAlert'
+import { AlertCategory } from 'components/Alerts/AlertsBanner'
+import { RobotFlotillaStatus } from 'models/Robot'
+import { useAlertContext } from './AlertContext'
+
+interface IDockContext {
+    DockStatus: boolean
+}
+
+interface Props {
+    children: React.ReactNode
+}
+
+const defaultEmergencyInterface = {
+    DockStatus: false,
+}
+
+export const EmergencyContext = createContext<IDockContext>(defaultEmergencyInterface)
```

Comment:
Why are some things called "Emergency", and others "Dock" in this file? The provider and exported context should at least have matching names